src/app/sign-up-get-starter-deso/sign-up-get-starter-deso.component.html

```diff
@@ -0,0 +1,182 @@
+    <div class="fs-18px">
+      Verify your phone number for free DESO. Your phone number will never be shared with anyone.
+    </div>
+
+    <form #f="ngForm" [formGroup]="phoneForm" class="form-group mt-15px">
+      <ngx-intl-tel-input
+        style="font-size: 15px !important"
+        [cssClass]="'form-control sign-up__phone-input'"
+        [separateDialCode]="true"
+        [enableAutoCountrySelect]="true"
+        [enablePlaceholder]="true"
+        [searchCountryFlag]="true"
+        [selectFirstCountry]="false"
+        [selectedCountryISO]="CountryISO.UnitedStates"
+        maxLength=15
+        [phoneValidation]="true"
+        (ngModelChange)="onPhoneNumberInputChanged()"
+        name="phone"
+        formControlName="phone"
+      ></ngx-intl-tel-input>
+
+      <div class="fc-red fs-15px">
+        <!-- TODO: kinda obnoxious how this displays an error as soon as I start typing-->
+        <div *ngIf="f.form.value.phone && f.form.controls.phone.invalid" class="mt-10px">
+          Please enter a valid phone number
+        </div>
+
+        <div *ngIf="sendPhoneNumberVerificationTextServerErrors.phoneNumberAlreadyInUse" class="mt-10px">
+          This phone number is being used by another account. Please use a different phone number or sign into your other
+          account.
+        </div>
+
+        <div *ngIf="sendPhoneNumberVerificationTextServerErrors.maxSendAttemptsReached" class="mt-10px">
+          You've requested too many verification codes. Please wait 10 minutes and try again.
+        </div>
+
+        <div *ngIf="sendPhoneNumberVerificationTextServerErrors.voipNumberNotAllowed" class="mt-10px">
+          This phone number is a VOIP number, which isn't allowed (to prevent spam). Please try again with a different
+          phone number.
+        </div>
+
+        <div *ngIf="sendPhoneNumberVerificationTextServerErrors.chineseNumberNotAllowed" class="mt-10px">
+          Currently we can't send text messages to China. If you have a non-Chinese number, please try that. If not, click
+          "skip" below. You can still use the site by buying DeSo.
+        </div>
+        <div *ngIf="sendPhoneNumberVerificationTextServerErrors.unknownError" class="mt-10px">
+          {{ sendPhoneNumberVerificationTextServerErrors.unknownError }}
+        </div>
+      </div>
+    </form>
+
+    <div class="fs-10px">
+      By proceeding, you agree to receive text messages from deso.org (standard rates apply)
+    </div>
+
+    <div class="d-flex mt-15px justify-content-end">
+      <div>
+        <a
+          *ngIf="displayForSignupFlow"
+          (click)="onSkipButtonClicked()"
+          class="btn btn-outline-primary font-weight-bold fs-15px"
+        >
+          Skip
+        </a>
+        <a
+          (click)="sendVerificationText()"
+          [ngClass]="{ disabled: !phoneForm.valid, 'btn-loading': sendingPhoneNumberVerificationText }"
+          class="btn btn-primary font-weight-bold fs-15px ml-10px"
```

Comment:
Use classes `button button-primary button-small` instead of `btn btn-primary` to match new identity theme